pkgs/tools/networking/bitmask-vpn/default.nix

```diff
@@ -0,0 +1,148 @@
+{ lib
+, fetchFromGitLab
+, mkDerivation
+, buildGoModule
+, wrapQtAppsHook
+, python3Packages
+, pkg-config
+, openvpn
+, cmake
+, qmake
+, which
+, iproute2
+, iptables
+, procps
+, qmltermwidget
+, qtbase
+, qtdeclarative
+, qtquickcontrols
+, qtquickcontrols2
+, qttools
+, provider ? "riseup"
+}:
+let
+  version = "0.21.6";
```

Comment:
latest seems to be 0.21.8 https://0xacab.org/leap/bitmask-vpn/-/blob/main/CHANGELOG

Reply:
@happysalada , indeed a changelog entry is there, and the milestone is past due, but there is no tag yet: https://0xacab.org/leap/bitmask-vpn/-/tags